### ISSUE TITLE
Load Graph: Name, role and state properties are not defined for 'full screen graph view' control present under 'Graph' tab section.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1060]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1248952

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Gremlin(graph)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to "New Graph" menu item and select it.
Navigate through all the controls under Add Graph blade and enter details and select ok button to create new 'Graph'.
Create 'Sample Graph under 'Gremlin  API' section and expand created graph control. 
Navigate through all the controls under "Gremlin API" Section and select "Graph" which appears after expending the items.
Navigate through all the controls like JSON query edit field type query and select Load Graph.
Navigate to 'New Vertex' menu item and fill the details under 'New Vertex' blade and select ok button to get the data available under Graph and JSON tab section.
Navigate through all the controls under JSON and Graph Tab items.
Verify the issue.

Actual Result:
Name, role and state properties are not defined for 'full screen graph view' control present under 'Graph' tab section due to this user won't be able to identify the control and won't be perform functionality over that control. 

Expected Result:
Name, role and state properties should be defined for 'full screen graph view' control present under 'Graph' tab section.
Name as "View graph in full screen", Role as 'button' and state(aria-expanded) should be defined properly for control.

![image](https://user-images.githubusercontent.com/8519407/132712470-77cb4bde-4ea7-4b1e-aa85-4ffff2d46aab.png)
